content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,63 @@
 # Roles and Responsibilities
 
 ## Team Organization
+Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the quiz at any given time.
 
 ### Quizzers
 
 ### Captains
+- Scorekeepers must be notified which quizzer is captain and co-captain before the quizzers take their seats to begin the quiz.
+- The team captain and co-captain may change during a given competition but not during a quiz.
+- Only the team captain or co-captain may challenge a decision or respond to a challenge.  If both the captain and co-captain have errored out, another quizzer can be designated by the coach to respond to or initiate challenges.   
 
 ### Coaches
 
 ## Quiz Officials
+- No comment other than “correct” or “incorrect” need be announced by the quiz officials in making a decision.  
```

Comment:
I put this at the Quiz Officials level, since it applies to both QM and AJ.

I fleshed out the language around explanations. It's reference A LOT, under both QM and AJ sections.